Source/Core/Core/IOS/ES/Formats.cpp

```diff
@@ -285,17 +285,17 @@ const std::vector<u8>& TicketReader::GetRawTicket() const
 
 std::vector<u8> TicketReader::GetRawTicketView(u32 ticket_num) const
 {
-  // A ticket view is composed of a view ID + part of a ticket starting from the ticket_id field.
+  // A ticket view is composed of a version + part of a ticket starting from the ticket_id field.
   const auto ticket_start = m_bytes.cbegin() + GetOffset() + sizeof(Ticket) * ticket_num;
   const auto view_start = ticket_start + offsetof(Ticket, ticket_id);
 
-  // Copy the view ID to the buffer.
-  std::vector<u8> view(sizeof(TicketView::view));
-  const u32 view_id = Common::swap32(ticket_num);
-  std::memcpy(view.data(), &view_id, sizeof(view_id));
+  // Copy the ticket version to the buffer (a single byte extended to 4).
+  std::vector<u8> view(sizeof(TicketView::version));
+  const u32 version = static_cast<u32>(m_bytes.at(offsetof(Ticket, version)));
```

Comment:
SGTM as long as the version value is correct according to the used endianness (i.e. if "extended to 4" means a single byte followed by 3 padding bytes and not a uint32_t in Big Endian).